src/frontend/web_application/src/components/BlockList/components/ItemContent/style.scss

```diff
@@ -0,0 +1,6 @@
+@import '../../../../styles/common';
+
+.m-item-content {
+  padding: $co-component__spacing 0;
+  border-bottom: 1px solid $co-color__bg__back;
```

Comment:
may be we should move the border to the `m-block-list__item` 
then remove completely the ItemContent component and let the responsibility of the padding to the developer. 

WDYT @myleneb ?